HTML/html_completions.py

```diff
@@ -365,4 +367,3 @@ def test_expand_tag_attributes(self):
         # should give just one completion, and suppress word based completion
         self.assertEqual(completion_list, [('td.class', '<td class="class">$1</td>$0')])
         self.assertEqual(flags, sublime.INHIBIT_WORD_COMPLETIONS | sublime.INHIBIT_EXPLICIT_COMPLETIONS)
-
```

Comment:
There were actually two newlines, that's just the superfluos one gone there. I have that enabled, because I don't want to mess with you vim users. :)